### ISSUE TITLE
toggleClass throws "is not a function"

### ISSUE THREAD

**Issue by typeoneerror** [Robdel12/DropKick#360]
### Version of Dropkick:

Version on http://dropkickjs.com/

### Expected Behavior

Multi-select should select items.

### Actual Behavior

Doesn't select, throws errors.

```
Uncaught TypeError: [((intermediate value)(intermediate value)(intermediate value) + "Class")] is not a function
    at Object.toggleClass (VM550 dropkick.js:1)
    at e.value (VM550 dropkick.js:1)
    at e.value (VM550 dropkick.js:1)
    at e.value (VM550 dropkick.js:1)
```

### Steps To Reproduce:
1. Click any item in multi-select example

### JSBin / Example repo / Failing test:

Pretty sure https://github.com/Robdel12/DropKick/blob/master/src/utils.js#L24 should be called on `this`.

**Comment by typeoneerror** [comment]
Failing test and experimental fix: https://github.com/Robdel12/DropKick/pull/361

**Comment by Robdel12** [comment]
This was closed with #361. Thanks @typeoneerror for the report AND the fix! :D
